README.adoc

```diff
@@ -238,10 +238,16 @@ fun `WHEN putting file on sftp THEN file is read and moved to a correct director
 ----
 
 === Absolute vs relative paths
-The SFTP Server Stub supports both absolute and relative paths (relative are interpreted as relative to user's
-home directory). However, in connection to flow testing you want to avoid using absolute paths, as the `readFiles`
-and `writeFiles` processors in Utilihive Flow Server interpret all paths as relative to user's home directory,
-whether they are prepended with `/` or not.
+The SFTP Server Stub supports both absolute and relative paths. Paths without `/`
+prefix are interpreted relative to `defaultDirectory` (`/`, unless overridden
```

Comment:
I agree that CIark should do copy edit on the docs, but I think we need to give him something that is a bit clearer to start with.  I find this whole part very hard to understand. I do not think that the term "default directory" will help our users understand what this setting is about. Its unclear to me why we are so averse to using language about this that is more in line with actual reality. We are simulating an SFTP server. Most SFTP servers will present you with a limited view of the file system where the path "/" is the top of the directory structure your sftp user has access to. So in our default setup you are logged in at "/" and can never access anything  above this position.
This means that relative paths and absolute paths are the same. I.e. "/files/in" and "files/in" points to the same place in the file system.

In some (I would argue, rare, cases - since I have never encountered this in any enterprise SFTP setup), you are actually not logged into the top of the file system  you have access to, but at some other place, say "/home/myuser". In that case relative paths and absolute paths are not the same. I.e. "/files/in" and "files/in" points to different places in in the file system ("/files/in" and "/home/myuser/files/in" respectively). All this seen from the point of view of an integation client where cd'ing around and CWD is not really the theme, more what directory structure you are logged into.

Reply:
Rewritten the way we agreed on Slack.